lib/Language/Haskell/Stylish/Step/ModuleHeader.hs

```diff
@@ -221,6 +257,11 @@ printMultiLineExportList conf (L srcLoc exportsWithComments) = do
     -- > xxxx( -- Some comment
     -- > xxxxyyfoo
     -- > xxxx) where
+
+    firstChar =
+      if openBracket conf == SameLine then " "
```

Comment:
I would suggest using:
```haskell
case openBracket conf of
  SameLine -> " "
  NextLine -> "("
```
which is going to stop compiling when a new `OpenBracket` option is added

Reply:
good one, thx!